README.md

```diff
@@ -55,6 +55,16 @@ symfonycasts_reset_password:
     enable_garbage_collection: true
 ```
 
+Production environments may require the default_uri to be defined in config/packages/routing.yaml to prevent the URI in the email to point to localhost.
```

Comment:
```suggestion
The production environment may require the `default_uri` to be defined in the `config/packages/routing.yaml` to prevent the URI in emails to point to localhost.
```